scripts/semantic-commits/get-binary-release-data.js

```diff
@@ -61,7 +122,7 @@ const getReleaseData = async (latestReleaseInfo) => {
   const prsInRelease = []
   const commits = []
 
-  await Promise.all(semanticCommits.map(async (semanticResult) => {
+  await Bluebird.each(semanticCommits, (async (semanticResult) => {
```

Comment:
This alone might prevent hitting rate limits since it makes the requests serial instead of racing them, but there's also the error handling just in case.